tools/dep_updaters/update-googletest.sh

```diff
@@ -86,4 +86,4 @@ echo "$NEW_GYP" >"$DEPS_DIR/googletest/googletest.gyp"
 # Update the version number on maintaining-dependencies.md
 # and print the new version as the last line of the script as we need
 # to add it to $GITHUB_ENV variable
-finalize_version_update "googletest" "$NEW_VERSION"
```

Comment:
Will this fail the commit title length check?

Reply:
It shouldn't (btw your comment is on the deleted line, which makes it harder to understand what we're talking about)